### ISSUE TITLE
Midi messages no longer passed through

### ISSUE THREAD

**Issue by InTheMorning** [jack-mixer/jack_mixer#73]
When controlling the mixer via midi, jack_mixer accepts new values, but doesn't forward them through midi out.  Controlling the mixer via its UI does output midi to reflect changes.

Shouldn't the mixer reflect its changes via midi out, regardless of the method that was used to make the change?

**Comment by SpotlightKid** [comment]
I believe this never worked correctly as you describe it. Actually while investigating this, I found deeper issues with the MIDI output.

Apparently the code for handling this predates versions of jack_mixer, which added MIDI control of balance, solo and mute, so it only sends out MIDI for volume changes, but uses a CC number corresponding to the channel index, not the correct CC number, which is assigned in the channel properties.

I created PR #76 to fix this (needs more testing before merging).

Sending out MIDI in reaction to changes triggered by MIDI input needs some investigation first, we need to be careful not to trigger a notification loop.

**Comment by SpotlightKid** [comment]
I merged #76 now, which at least fixes MIDI CC output for changes originating from the UI.

**Comment by SpotlightKid** [comment]
@InTheMorning PR #95 should fix this. Would you be able to test it?

```console
git clone https://github.com/SpotlightKid/jack_mixer.git
cd jack_mixer
git checkout enhancement/73-midi-output-from-input
./autogen,sh --prefix=/usr
make
PYTHONPATH=".libs:$PYTHONPATH" python jack_mixer.py
```

See comment on #95 for things to test.
